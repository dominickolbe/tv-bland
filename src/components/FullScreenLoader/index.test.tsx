import { shallow } from "enzyme";
import React from "react";
import { FullScreenLoader } from "./";

describe("<FullScreenLoader />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(<FullScreenLoader />);
    expect(wrapper).toMatchSnapshot();
  });
});
