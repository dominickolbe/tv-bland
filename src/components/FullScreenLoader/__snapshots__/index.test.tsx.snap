// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FullScreenLoader /> renders snapshots 1`] = `
<WithStyles(ForwardRef(Backdrop))
  open={true}
>
  <WithStyles(ForwardRef(CircularProgress))
    color="primary"
  />
</WithStyles(ForwardRef(Backdrop))>
`;
